pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -65,8 +65,8 @@ type Bootstrapper struct {
 }
 
 // NewBootstrapper creates a new kubeadm.Bootstrapper
-func NewBootstrapper(api libmachine.API) (*Bootstrapper, error) {
-	name := viper.GetString(config.MachineProfile)
+func NewBootstrapper(api libmachine.API, nodeName string) (*Bootstrapper, error) {
```

Comment:
I'm a bit confused: what and why is nodeName here? 

It feels like the bootstrapper shouldn't need to know about individual nodes names. It's unclear to me which one I would pass here.